contrib/IECoreUSD/test/IECoreUSD/data/customAttribute.usda

```diff
@@ -3,8 +3,27 @@
 def Sphere "sphere"
 {
 	custom string customNotNamespaced = "red"
-	custom string customNamespaced:test = "green"
+	custom string primvars:user:test = "green"
+	custom string studio:foo = "brown"
 	string notNamespaced = "blue"
 	string namespaced:test = "purple"
 	double radius = 10
+	custom string primvars:user:notAConstantPrimVar = "pink"(
+		IECOREUSD_CONSTANT_PRIMITIVE_VARIABLE = false
+	)
+}
+def Xform "a"
+{
+	custom string primvars:user:foo = "yellow"
+	def Sphere "b"
+	{
+		custom string primvars:bar = "black"(
+			IECOREUSD_CONSTANT_PRIMITIVE_VARIABLE = true
+		)
+		custom string primvars:user:baz = "white"
```

Comment:
I've not found the correct spot in the code to make this comment, so I've just stuck it here so it can be (eventually) resolved.

If I add `custom string user:bongo = "purple"` then get the following error when loading into Gaffer (and the attribute does not show up):

`WARNING : SceneReader::computeAttributes : Failed to load attribute "user:bongo" at location "/a/b"`

Reply:
addressed here 9aa9e274cc494f73274ac80b67201b139f7474ff